instruqt/closing-information-gap/04-adding-to-the-chart/assignment.md

```diff
@@ -101,25 +101,19 @@ definition.
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "common.names.fullname" . }}-support-bundle
+  name: {{ include "slackernews.name" . }}-support-bundle
   namespace: {{ .Release.Namespace | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
+  labels: {{- include "slackernews.labels" . | nindent 4 }}
     app.kubernetes.io/component: troubleshoot
     troubleshoot.sh/kind: support-bundle
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-  {{- end }}
 type: Opaque
 stringData:
   support-bundle-spec: |
 ```
 
 Remmeber that the support bundle must be at a key named `support-bundle-spec`
 to be found when generating a support bundle. Open the file
-`harbor-support-bundle.yaml` and copy/paste its content into the secret
+`slackernews-support-bundle.yaml` and copy/paste its content into the secret
```

Comment:
They don't expose a config, but I'm wondering if they changed the default config since I never noticed this in earlier runs through.

Reply:
I've created a [shortcut](https://app.shortcut.com/replicated/story/104084/use-four-spaces-in-yaml-code-fragments-to-simplify-copy-paste) for this one. I'm going to take it up as a chore since it cuts across multiple labs and isn't directly related to migrating these from Harbor to Slackernews.